src/status_im/contexts/shell/activity_center/events.cljs

```diff
@@ -492,6 +495,14 @@
              {:error error
               :event :activity-center/mark-as-seen}))
 
+(re-frame/reg-fx :activity-center.notifications/fetch-unread-count
```

Comment:
Similar to another comment I made, effects should go in a `effects.cljs` file and have a prefix `:effects`. Oh hey, personally I don't like the word `effects` as a prefix because it's too long, but it's the current convention.